examples/ffva/src/app_conf.h

```diff
@@ -50,7 +50,7 @@
 
 /* Enable audio response output */
 #ifndef appconfAUDIO_PLAYBACK_ENABLED
-#define appconfAUDIO_PLAYBACK_ENABLED           1
+#define appconfAUDIO_PLAYBACK_ENABLED           0
```

Comment:
From the file name, I assume that this file contains the FFVA application configuration constants.  What is the reason for changing this constant from enabled to disabled?  The same question applies to the `appconfINTENT_RAW_OUTPUT` and `appconfINTENT_I2C_OUTPUT_ENABLED` constants.

Reply:
appconfAUDIO_PLAYBACK_ENABLED   requires a TX I2S channels, we don't have them when we use the I2S audio as input to the intent engine. I will add a check that appconfAUDIO_PLAYBACK_ENABLED    and appconfUSE_I2S_INPUT cannot be enabled in the same build.

appconfINTENT_RAW_OUTPUT enables dumping all the audio samples received by the intent engine via UART interface. If I enable it, the intent engine stops working because it runs out of MIPS. I preferred to disable it for now. @mbanth, I can create a story to investigate the use of that option.

Reply:
I'm unclear if the direct I2S to Cyberon engine changes replace the previous FFVA example or if, in effect, a 2nd FFVA example has been added.  If a 2nd example has been added, why are the configuration changes being made in this file rather than in another app_conf.h file?  If a 2nd example hasn't been created, i.e., the changes replace the previous FFVA example, who is a customer suppose to use the old FFVA?